xrtl/gfx/context.h

```diff
@@ -44,6 +44,74 @@
 namespace xrtl {
 namespace gfx {
 
+// Defines a ReadBufferData buffer region.
+struct ReadBufferRegion {
+  ReadBufferRegion() = default;
+  ReadBufferRegion(size_t source_offset, void* target_data,
+                   size_t target_data_length)
+      : source_offset(source_offset),
+        target_data(target_data),
+        target_data_length(target_data_length) {}
+
+  // Layer range in the source buffer to read the data from.
```

Comment:
Seems this should explain what the offset is (and below), this isn't a Image::LayerRange.

Reply:
copypasta!